tests/Microsoft.Identity.Test.Unit/BrokerTests/WamMsaPassthroughTests.cs

```diff
@@ -89,7 +89,7 @@ public async Task FetchTransferTokenAsync()
                    });
 
                 // Act
-                var transferToken = await _msaPassthroughHandler.TryFetchTransferTokenAsync(requestParams, msaProvider)
+                var transferToken = await _msaPassthroughHandler.TryFetchTransferTokenInteractiveAsync(requestParams, msaProvider)
```

Comment:
Should there also be tests for `TryFetchTransferSilentDefaultAccountAsync` and `TryFetchTransferTokenSilentlyAsync`?